pages/conference-day.tsx

```diff
@@ -0,0 +1,233 @@
+import React from 'react'
+import { GetServerSideProps, NextPage } from 'next'
+import dynamic from 'next/dynamic'
+import Link from 'next/link'
+import { format } from 'date-fns'
+import { useConfig } from 'Context/Config'
+import { Session } from 'config/types'
+import { Main } from 'layouts/main'
+import { getCommonServerSideProps } from 'components/utils/getCommonServerSideProps'
+import { fetchSessions } from 'components/utils/useSessions'
+import { useSessionGroups } from 'components/utils/useSessionGroups'
+import { StyledList, Text } from 'components/global/text'
+import { SafeLink } from 'components/global/safeLink'
+import { roomLocations } from 'components/venueMapData'
+import Image from 'next/image'
+
+const VenueMap = dynamic(() => import('components/venueMap'), { ssr: false })
+
+interface ConferencePageProps {
+  sessions?: Session[]
+  sessionId?: string
+}
+
+const ConferenceDayPage: NextPage<ConferencePageProps> = ({ sessions }) => {
+  const { conference, dates } = useConfig()
+  if (dates.IsInProgress) {
+    // dummy
+  }
+
+  const { currentSessionGroup } = useSessionGroups(sessions)
+
+  if (currentSessionGroup && currentSessionGroup.sessions.length > 0) {
+    // NB: This is quite brittle, as it assumes that the list of current sessions is returned in exactly the same order as the definition of rooms
+    currentSessionGroup.sessions.map(function (session: Session, i) {
```

Comment:
I wonder if you get the same problem I had with the multi-session talk here. Worth trying it out with the testing control setting time to 3pm?

Reply:
Yes, please test on local if you get a chance.

Reply:
@bplowry So this does break, given the reliance on an exactly-ordered list of sessions to map to rooms. The inclusive teams panel should be in Bilya, but it's appearing against Kwonding:

![image](https://user-images.githubusercontent.com/818666/189044379-493c4229-3656-44ef-b83d-0a718019b7e9.png)
 
I'm not sure I'll be able to find a proper fix in time, so please let me know if you can help. Alternatively maybe we can add a bit of a hardcoded hack just for this time...